poetry.lock

```diff
@@ -1,10 +1,16 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
I'm assuming you have installed Poetry version `1.3.x`, while in the CI and for Docker releases we use `1.2.2`, could I please ask you to downgrade locally and regenerate the `poetry.lock` file 🙏🏻